src/components/AsideHeader/components/CompositeBar/Item/Item.module.scss

```
@@ -281,6 +375,19 @@ $actionLeftRightMargin: 10px;
         }
     }
 
+    &_menu-group-nested {
+        // Match group header title inset: icon slot + title gap.
+        padding-inline-start: calc(
+            var(--gn-aside-header-item-height, #{variables.$item-height}) +
```

Stylelint (17.13.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)